### PR TITLE
33 API Create Account Functionality

### LINKED ISSUE #33: API Create Account Functionality
API Post Request for creating account

### PR DESCRIPTION
**Pull request by Jackson-Wheeler**
resolves #33

### CODE REVIEW

File: server/server_code/database.py [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 class Database:
   def __init__(self, path_to_db):
     print(f"connecting to database at '{path_to_db}'")
-    connection = sqlite3.connect(path_to_db)
-    self.cursor = connection.cursor()
+    self.connection = sqlite3.connect(path_to_db)
+    self.cursor = self.connection.cursor()
 
   # Return a JSON object of users
   def get_users(self):
```
Comment (sbenetz) [inline server/server_code/database.py:24]: Does this need to be SELECT instead as well?

**Review by sbenetz — COMMENTED** [review]
This is going to be cool to see happen from http request. Hopefully Yuancheng can get an API up on the phone side soon to test it real time. Good work!

File: server/server_code/database.py [hunk 2]
```diff
@@ -14,5 +27,58 @@ def get_users(self):
```
Comment (sbenetz) [inline server/server_code/database.py:27]: Does the above need to be CAPS SELECT as well?

**Review by sbenetz — APPROVED** [review]

File: server/server_code/database.py (hunk 2 above)
Comment (Jackson-Wheeler) [reply server/server_code/database.py:27]: Hey Shane, what do you mean by CAPS SELECT?

**Pull request merged**